snapcraft/internal/libraries.py

```diff
@@ -102,3 +103,18 @@ def get_dependencies(elf):
     libs = [l for l in ldd_out if not os.path.basename(l) in system_libs]
 
     return libs
+
+
+_libc_lib_list = None
```

Comment:
on second thought, this should be in repo since it call `dpkg` and will break the crossdistro story (or cause cyclic imports which is also an unhappy story).